DOC: Add install note re: advanced drivers on conda-forge (#474)
geopandas/pyogrio: docs/source/install.md

raw I/O support and `geopandas` and their dependencies for GeoDataFrame
I/O support.

To enable additional GDAL divers, you may need to install them directly:

> -   LIBKML: [libkml](https://anaconda.org/conda-forge/libkml)

Reviewer: Based on the list of available plugins as conda packages (https://gdal.org/en/latest/download.html#conda) I would assume that KML is still built-in with libgdal-core?

Reviewer: And the same for spatialite I assume

Reviewer: It would still be good to mention this (the fact that some drivers need an additional install), but looking at that list, the only three vector drivers that get installed separately as plugin are `libgdal-arrow-parquet`, `libgdal-pg` and `libgdal-xls`, I think

raw I/O support and `geopandas` and their dependencies for GeoDataFrame
I/O support.

> By default, the `GDAL` package on conda-forge currently includes LIBKML and
> Spatialite. However, you may need to install additional drivers by installing
> the associated [conda-forge package](https://gdal.org/en/latest/download.html#conda):

Reviewer 2: ```suggestion
By default, the `GDAL` package on conda-forge already supports a wide range of vector formats. If needed, you can install additional drivers by installing
the associated [conda-forge package](https://gdal.org/en/latest/download.html#conda). Currently, the following packages to install extra vector drivers are available:
```

> -   `libgdal-arrow-parquet`
> -   `libgdal-pg`
> -   `libgdal-xls`

Reviewer: ```suggestion
-   `libgdal-arrow-parquet` ((Geo)Parquet and (Geo)Arrow IPC)
-   `libgdal-pg` (PostgreSQL / PostGIS)
-   `libgdal-xls` (XLS - MS Excel format)
```

(for `pg` I think it is certainly valuable to add the full name for clarity, the others might be clear already)

source distribution because the available wheels are not compatible with your
platform.

The binary wheels available on PyPI include the core GDAL drivers (GeoJSON,
ESRI Shapefile, GPKG, FGB, OpenFileGDB, etc) but do not include more advanced
drivers such as LIBKML and Spatialite. Use conda-forge to install pyogrio and

> then install these additional drivers directly if needed.

Reviewer 2: I would still understand from this that LIBKML and spatialite would need installing extra things when using conda-forge?

Author: Should we include a "see above" to make that a bit more clear?  I was trying to get at that with the "if needed", but maybe it is still confusing as written?

Reviewer 2: Maybe something like this?

```suggestion
drivers such as LIBKML and Spatialite. If you need such drivers, it is recommended to use conda-forge to install pyogrio as explained above.
```
